mapper.go

```diff
@@ -58,7 +59,7 @@ func (m *metricMapper) initFromString(fileContents string) error {
 
 		switch state {
 		case SEARCHING:
-			if line == "" {
+			if line == "" || line[0] == COMMENT_CHAR {
```

Comment:
Comments everywhere make sense, are quite more difficult to implement. `name="foo#bar" # comment` would need to be handled as well.